src/arrays/programs_involving_multidimensional_data/easy/q2/LineUps.java

```diff
@@ -0,0 +1,25 @@
+package arrays.programs_involving_multidimensional_data.easy.q2;
+
+/*
+A 2D array is used to keep track of the order of the people who appeared in line-ups. In particular, each row represents
```

Comment:
Change question text to:
`Create a Java program that uses a 2D array to keep track of the order of people who appear in a line. In particular, each row represents the names of those who lined up for a musical event. Then, your program should go through all the lines and check if the person in the last spot is the same. If it is the same, the program should print "Match". For example, if the name "Sam" is last in two of the lines, your program should print "Match" twice.`